### PR TITLE
Tim's Hard-Del Crusade: Radial Menu Center Buttons

### PR DESCRIPTION
**Pull request by DatBoiTim**
Prevents Hard-Del

## About The Pull Request
Prevents hard-del of  `/atom/movable/screen/radial/center` by clearing out an extant reference.

## Why It's Good For The Game
Hard-dels take a lot of time.

## Changelog
:cl: DatBoiTim
fix: Fixed Hard-Del of Radial Menu Center Button
/:cl:

### CODE REVIEW

File: code/_onclick/hud/radial.dm [hunk 1]
```diff
@@ -67,6 +67,11 @@ GLOBAL_LIST_EMPTY(radial_menus)
 	if(usr.client == parent.current_user)
 		parent.finished = TRUE
 
+/atom/movable/screen/radial/center/Destroy()
+	.=..()
```
Comment (PowerfulBacon) [inline code/_onclick/hud/radial.dm]:
```suggestion
	. = ..()
```

**Review by PowerfulBacon — DISMISSED** [review]

**Comment by DatBoiTim** [comment]
👍

File: code/_onclick/hud/radial.dm [hunk 2]
```diff
@@ -67,6 +67,11 @@ GLOBAL_LIST_EMPTY(radial_menus)
 	if(usr.client == parent.current_user)
 		parent.finished = TRUE
 
+/atom/movable/screen/radial/center/Destroy()
+	. = ..()
+	if(parent)
+		parent.close_button = null
```
Comment (PowerfulBacon) [inline code/_onclick/hud/radial.dm:72]: Do it before the destroy behaviour in ..()

Comment (DatBoiTim) [reply code/_onclick/hud/radial.dm:72]: As in do it under the parent proc or put . = ..() at the end?

**Comment by DatBoiTim** [comment]
Should not have done a webedit

**Comment by Crossedfall** [comment]
Pull master to fix the linters

File: code/_onclick/hud/radial.dm (hunk 2 above)
Comment (Archanial) [reply code/_onclick/hud/radial.dm:72]: return ..()

Comment (DatBoiTim) [reply code/_onclick/hud/radial.dm:72]: f5b89c8 Should adequately address this concern.

**Review by Archanial — APPROVED** [review]

**Pull request merged**